test/unit/stats_test.lua

```diff
@@ -0,0 +1,79 @@
+local expirationd = require("expirationd")
+local fiber = require("fiber")
+local t = require("luatest")
+local g = t.group("stats")
+
+local helpers = require("test.helper")
+
+g.before_all(function()
+    helpers.init_spaces(g)
+end)
+
+g.after_each(function()
+    helpers.truncate_spaces(g)
+end)
+
+function g.test_stats_basic()
```

Comment:
I think a general approach to such metrics should be following:
- Check that counter is 0
- Perform some actions
- Check that counter increased.

Currently there is an issue. Stats could have initially expected value and then test doesn't perform any action and finally we have expected result.

Reply:
I check initial counter's values in a first testcase. I don't see any reasons to check this in each testcase.